<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>800</width>
      <height>631</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget#stackedWidget{
background-color: qlineargradient(spread:pad, x1:0.107, y1:0.119455, x2:1, y2:1, stop:0 rgba(125, 93, 255, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255); font-size:80pt;
font: 8pt &quot;MV Boli&quot;;
font: 75 30pt &quot;Bahnschrift SemiCondensed&quot;;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
font-size: 18px;
background:transparent;
border:none;
color:#717072

}</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
font-size: 18px;
background:transparent;
border:none;
color:#717072

}</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enterButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>380</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  display: inline-block;
  border-radius: 4px;
  background-color:#ffffff;
  border: none;
  color: rgb(143, 143, 143);
  text-align: center;
  font-size: 28px;
  padding: 20px;
  width: 200px;
  transition: all 0.5s;
  cursor: pointer;
  margin: 5px;
}

QPushButton span {
  cursor: pointer;
  display: inline-block;
  position: relative;
  transition: 0.5s;
}

.button span:after {
  content: '\00bb';
  position: absolute;
  opacity: 0;
  top: 0;
  right: -20px;
  transition: 0.5s;
}

QPushButton:hover span {
  padding-right: 25px;
}

QPushButton:hover span:after {
  opacity: 1;
  right: 0;
}</string>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QLabel" name="invalid">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>270</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Invalid email or password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>enterButton</zorder>
   <zorder>password</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>email</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
